### DISCUSSION TITLE
Removal of solutions tied an account

### DISCUSSION THREAD

**Discussion by fikotta** [codewars/codewars.com#2735] [Q&A]
Could a solution be removed and honor deducted as a 1 time thing? Asking for me.

**Comment by fikotta** [comment]
if so, my most recent solution please.

**Reply by hobovsky** [reply]
You want your solution to "Cut the cake" to be invalidated, am i getting this right?

**Reply by fikotta** [reply]
Yes please, a misclick happened. Don't want my account tainted.

**Reply by hobovsky** [reply]
Honor withdrawn per user's request.

**Reply by fikotta** [reply]
@hobovsky Thank you very much.

**Comment by hobovsky** [comment]
I am not sure what exactly you are asking, but as far as i can understand the question, answer would be 'Yes'.

This type of questions is the easiest to handle if you come with more details to `#support-requests` channel of Codewars Discord. Is this OK for you?

**Reply by fikotta** [reply]
Hardly use Discord myself, and seeing you helped me on github discussions once, came here first.

**Reply by hobovsky** [reply]
It's OK, I just was not sure if I understand your question correctly, and for me it's faster to communicate over a chat.  
But now I got what you need, so it's fine.
